publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategy.java

```diff
@@ -38,7 +38,7 @@ public PublicationPermissionStrategy(
         this.publication = publication;
         this.grantStrategies = Set.of(
             new EditorPermissionStrategy(publication, userInstance, resourceService),
-            new CuratorPermissionStrategy(publication, userInstance, resourceService),
+            new PublishingCuratorPermissionStrategy(publication, userInstance, resourceService),
```

Comment:
Not sure this is the right thing to do. By introducing "PublishingCurator", we lack "SupprotCurator", "DoiCurator", "NviCurator" and other type of curators. I think CuratorPermissionStrategy is general permission check for any type of curator.

Reply:
hmm yeah I agree! Will change back for this PR, but I think it would be better to separate the concerns in different strategies somehow. Our code today tries to infer roles based on specific access grants and mixes different grants in same strategy.